core/src/main/scala/cell/Cell.scala

```diff
@@ -90,6 +94,9 @@ trait Cell[K <: Key[V], V] {
   // Only used in tests.
   private[cell] def waitUntilNoNextDeps(): Unit
 
+  private[cell] def tasksActive: Boolean
```

Comment:
The `tasksActive` method is not referentially transparent (its result depends on state). Therefore, its name should have parentheses: `tasksActive()`